### ISSUE TITLE
ChainMgr is unable to find iptables executable

### ISSUE THREAD

**Issue by pyllyukko** [mrash/psad#28]
On a Slackware system, I get this:
```
[*] Could not find/execute iptables, specify path via _iptables
 at /usr/lib/psad/IPTables/ChainMgr.pm line 37.
```

Would it be possible for ChainMgr to check the availability of ```iptables``` & ```ip6tables``` also from ```/usr/sbin``` or use the ```iptablesCmd``` & ```ip6tablesCmd``` variables from ```/etc/psad/psad.conf```?

**Comment by mrash** [comment]
Yes, thanks for the bug report. I'll get this fixed.

**Comment by mrash** [comment]
This has been fixed with IPTables::Parse 1.6.1, and psad now relies on it for the proper path (current sources in git master).

**Comment by mrash** [comment]
psad-2.4.2 has been released which contains this fix: http://www.cipherdyne.com/psad/download/
